skimage/restoration/_denoise_cy.pyx

```diff
@@ -176,22 +133,21 @@ def _denoise_tv_bregman(image, double weight, int max_iter, double eps,
         Py_ssize_t total = rows * cols * dims
 
     shape_ext = (rows2, cols2, dims)
-    u = np.zeros(shape_ext, dtype=np.double)
+    u = out.copy()
```

Comment:
you should explain what this is. Can you call np.zeros_like instead?

Reply:
I can definitely add a comment here